### ISSUE TITLE
Why php-feed-writer is a "project" composer type

### ISSUE THREAD

**Issue by pH-7** [lukaswhite/php-feed-writer#5]
Hello Lukas!

I believe this library should be a "library" type in your [composer.json file](https://github.com/lukaswhite/php-feed-writer/blob/master/composer.json#L4), and not a "project type. Because of this wrong type, I get the following error on composer v2 since it is not a library.

>  Could not find a matching version of package lukaswhitephp-feed-writer.

Finally, on packages.org, it shows `composer create-project ` as you mentioned `"type": "project"` in your composer.json file.

<img width="605" alt="Screenshot 2021-12-22 at 20 32 07" src="https://user-images.githubusercontent.com/1325411/147074648-e48a53f7-d977-4cfb-a89c-b11fb20641fb.png">

As soon as you change from `"type": "project",` to `"type": "library",`, packages.org will display `composer require` on [your library packagist page](https://packagist.org/packages/lukaswhite/php-feed-writer).

**Comment by pH-7** [comment]
FYI @lukaswhite, I created a PR to do so. https://github.com/lukaswhite/php-feed-writer/pull/6

**Comment by pH-7** [comment]
Let me know when you can check out the [PR](https://github.com/lukaswhite/php-feed-writer/pull/6), @lukaswhite 😃

**Comment by pH-7** [comment]
> Let me know when you can check out the [PR](https://github.com/lukaswhite/php-feed-writer/pull/6), @lukaswhite 😃

👋 @lukaswhite 🙂

**Comment by sudofox** [comment]
Is this project still maintained? I was looking at it as something I might be able to use

**Comment by pH-7** [comment]
@sudofox Well, it doesn't look to be maintained anymore... as my PR and this issue didn't get any replies from its author @lukaswhite, and the last update was made two years ago.

**Comment by pH-7** [comment]
@sudofox You could still create a fork from this one though, and build from there...

**Comment by lukaswhite** [comment]
Yes, it's still maintained. I hadn't seen this issue, it's been a tumultuous couple of years. I've been using it for a while with no problems, so there hasn't seemed much need to do anything on it - though a PHP8.1 version might be a good idea.

@pH-7 I'll get that PR merged this week.
@sudofox Any specific issues, or just checking?

**Comment by sudofox** [comment]
@lukaswhite I was just checking -- been thinking about adding RSS/Atom feed functionality to a small website of mine.
